lightwood/encoders/numeric/ts_numeric.py

```diff
@@ -37,7 +37,7 @@ def encode(self, data, extra_data=None):
                 except:
                     real = None
             if self.is_target:
-                vector = [0] * 3
+                vector = [0] * 2
```

Comment:
Why did we change the list size from 3 to 2?

Reply:
It's something that should have been done before: for TS numerical tasks the log target value (last element in the encoded vector) is not used, and leaving it as part of the vector unnecessarily complicates the optimization procedure.